src/main/fc/fc_tasks.c

```diff
@@ -204,20 +188,12 @@ static void taskUpdateBaro(timeUs_t currentTimeUs)
 }
 #endif
 
-#if defined(USE_BARO) || defined(USE_RANGEFINDER)
+#if defined(USE_BARO) || defined(USE_GPS)
 static void taskCalculateAltitude(timeUs_t currentTimeUs)
 {
-    if (false
-#if defined(USE_BARO)
-        || (sensors(SENSOR_BARO) && isBaroReady())
-#endif
-#if defined(USE_RANGEFINDER)
-        || sensors(SENSOR_RANGEFINDER)
-#endif
-        ) {
-        calculateEstimatedAltitude(currentTimeUs);
-    }}
-#endif // USE_BARO || USE_RANGEFINDER
+    calculateEstimatedAltitude(currentTimeUs);
```

Comment:
This also doesnt look right. This is missing the checks if the senors are activated

Reply:
Maybe they have been moved into `calculateEstimatedAltitude`?

Reply:
My understanding is that this task would not be enabled if the sensor was not activated.  I'm definitely not 100% positive on this though.